Mentoring-Program-Guide.md

```diff
@@ -341,3 +341,12 @@ A few things that you might want to do are:
 - Don't forget using [GSoC] or [Outreachy].
 
 - Don't forget using [RFC] or [WIP], if it's relevant.
+
+# Conclusion
+
+Hope you got an idea about how things usually go during the internship.
```

Comment:
Maybe "internship/GSoC/mentoring period" as "internship" is for Outreachy only. In general it was not easy when writing this and other such docs to make sure the terms would work for Outreachy, GSoC and possibly other future programs. Maybe somewhere we should have some kind of internal docs/dictionary listing the terms we should use.

Reply:
> Maybe somewhere we should have some kind of internal docs/dictionary listing the terms we should use.

Yeah. That would be good to have. I'll try and see if I could find the time to come up with this one.